kong/cmd/utils/prefix_handler.lua

```diff
@@ -40,33 +42,48 @@ client:request { \
 ]]
 
 local resty_bin_name = "resty"
+local resty_version_pattern = "nginx[^\n]-openresty[^\n]-([%d%.]+)"
+local resty_compatible = version.set(meta._DEPENDENCIES.nginx_from, meta._DEPENDENCIES.nginx_to)
 local resty_search_paths = {
   "/usr/local/openresty/bin",
   ""
 }
 
+local function is_openresty(bin_path)
+  local cmd = fmt("%s -V", bin_path)
+  local ok, _, _, stderr = pl_utils.executeex(cmd)
+  log.debug("%s: '%s'", cmd, stderr:sub(1, -2))
```

Comment:
`stderr:sub(1, -2)` prints too much information. Only the first line ('resty 0.12') should be relevant here (or the second one if we stick to checking the OpenResty version instead of resty-cli).